library/src/components/layouts/CircleLayout.tsx

```diff
@@ -18,7 +18,7 @@ function calculateLayout(
   elements: Node[],
   passedOptions: Partial<CircleLayoutOptions> = defaultOptions,
 ): FlowElement[] {
-  // const options = { ...defaultOptions, ...passedOptions };
+ 
```

Comment:
Think the linter complains that there are an empty line here 🙂

Reply:
So i make a PR after fixing it right ?